packages/shared/src/components/squads/SquadPageHeader.tsx

```diff
@@ -1,45 +1,84 @@
 import React, { ReactElement } from 'react';
+import classNames from 'classnames';
 import { Squad, SquadMember } from '../../graphql/squads';
 import { SquadHeaderBar } from './SquadHeaderBar';
 import { SquadImage } from './SquadImage';
 import EnableNotification from '../notifications/EnableNotification';
 import { NotificationPromptSource } from '../../hooks/useEnableNotification';
+import { FlexCol } from '../utilities';
+import SquadMemberShortList from './SquadMemberShortList';
+import useSidebarRendered from '../../hooks/useSidebarRendered';
+import SharePostBar from './SharePostBar';
 
 type SquadPageHeaderProps = {
   squad: Squad;
   members: SquadMember[];
   onNewSquadPost: () => void;
 };
 
+const MAX_WIDTH = 'laptop:max-w-[38.5rem]';
+
 export function SquadPageHeader({
   squad,
   members,
   onNewSquadPost,
 }: SquadPageHeaderProps): ReactElement {
+  const { sidebarRendered } = useSidebarRendered();
+
   return (
-    <section className="flex flex-col items-center px-6 pb-0 tablet:pb-10 mb-6 w-full tablet:border-b min-h-20 border-theme-divider-tertiary">
-      <SquadImage
-        className="mt-4 w-16 tablet:w-24 h-16 tablet:h-24"
-        {...squad}
-      />
-      <h3 className="mt-4 tablet:mt-6 font-bold typo-title2">{squad.name}</h3>
-      <h4 className="mt-1 tablet:mt-2 typo-body text-theme-label-tertiary">
-        @{squad.handle}
-      </h4>
-      <p className="mt-5 tablet:mt-4 w-full text-center typo-body text-theme-label-secondary max-w-[42rem]">
+    <FlexCol className="relative items-center laptop:items-start px-6 pb-20 tablet:pb-20 laptop:pb-14 mb-6 w-full tablet:border-b laptop:px-[4.5rem] min-h-20 border-theme-divider-tertiary">
```

Comment:
Note in the design the page has less top margin then on desktop (not sure if it's intentional)

Reply:
This is handled on the `/[handle]/index` page.